docs/continuous_integration.rst

```diff
@@ -0,0 +1,60 @@
+.. _ci_tests:
+
+Continuous Integration
+===============================
+
+The SecureDrop project uses `GitHub Actions <https://docs.github.com/en/actions>`_ for
+running automated continuous integration on code changes. You can get an overview of what
+each project does by reviewing the ``Makefile`` and files in the ``.github/workflows`` folder.
+
+Basics
+------
+
+Our CI runs a mixture of linters, tests and build processes to validate code submissions.
+
+Most tasks have a corresponding ``make`` target that will run the same command locally. Common
+targets across all our projects include:
+
+* ``make lint``: run linters
+* ``make fix``: apply automated fixes from formatters and linters
+* ``make test``: run automated tests
+
+In CI, these are run in a container using the corresponding Linux distribution (e.g. Debian or Fedora),
+which can also be used to reproduce CI results locally. Some projects, like ``securedrop`` (server) and
```

Comment:
Are these make targets used directly in the CI itself, or do they replicate the CI commands? If they are maintained separately, is it worth calling that out just in case there's a desync in the future and the CI ends up running different commands than the makefile?

Reply:
In nearly all cases we run the make targets in CI itself.